openfisca_us/variables/gov/states/il/tax/income/il_use_tax.py

```diff
@@ -0,0 +1,15 @@
+from openfisca_us.model_api import *
+
+
+class il_use_tax(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "IL use tax"
+    unit = USD
+    definition_period = YEAR
+    reference = ""
+
+    def formula(tax_unit, period, parameters):
+        agi = tax_unit("federal_agi", period)
```

Comment:
```suggestion
        # Tiered amount if income below a threshold, otherwise a percentage of AGI.
        agi = tax_unit("federal_agi", period)
```